Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth
# Artifacts as Reward for Society Quests

#### Changelog:
- Artifact gamerules now belong to separate group.
- Added chance to receive generic artifact after completing a society quest. Added society artifact rewards gamerule to regulate this.
- Characters with higher title tiers will have increased chance to receive high tier artifacts, while characters with lower title tiers will have increased chance to receive low tier artifacts on gamestart. Added gamestart artifacts rarity gamerule to regulate this.
- Various factors, like culture, affect chances of getting specific types of artifacts. Added artifacts preferences gamerule to regulate this.


#### Developer changelog:
- Reworked effect that spawns artifact.
- Renamed RULE_INVADERS_STRENGTH text lines in localization file, since it used not only for that gamerule


#### How to test:
- Added `test.44`&`test.45` events. You can use options tooltips to test artifact rarity and preferences.
- You also may want to check [design document](https://docs.google.com/spreadsheets/d/1x0bDnGuc2NHGw2iRAzru9CxtG8S-3nIZZdnBTc0Us8A/edit#gid=1500269219&range=1:24)
